notebooks/223-text-prediction/223-text-prediction.ipynb

```diff
@@ -0,0 +1,773 @@
+{
```

Comment:
<p>I think this part should be inside Conversation chapter and for consistency please provide some chapter "Inference with PersonaGPT" or accept to run ii in thext prediction mode too (it can work in this mode too, is not it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I did not add the other models for conversation because when i tried to use them for conversation they didn't give good outputs, because they have not been trained on conversational data and similarly PersonaGPT doesnt give good outputs on Text Continuation since it was trained on Chat Data. I have incorporated the other fix.</p>

Reply:
I think we need some explanation on description level  "if you use person gpt please skip this step" and som consistency in sections naming (I mean, you highlight that this section for GPT2 and GPT-Neo but without the whole notebook idea understanding it is difficult to find where inference part for PersonaGPT should be run, that is why I ask some consistency in chapters naming)